lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -212,25 +212,64 @@ describe('CategoryRenderer', () => {
         'no manual description');
   });
 
-  it('renders not applicable audits if the category contains them', () => {
-    const a11yCategory = sampleResults.categories.accessibility;
-    const categoryDOM = renderer.render(a11yCategory, sampleResults.categoryGroups);
-    assert.ok(categoryDOM.querySelector(
+  describe('categories with not applicable audits', () => {
+    let a11yCategory;
+
+    beforeEach(()=> {
+      a11yCategory = JSON.parse(JSON.stringify(sampleResults.categories.accessibility));
```

Comment:
Is there a better way than JSON deep cloning to prevent altering the ```sampleResults``` object across tests?

I think some kind of clone is necessary because I need to alter some of the object's attributes, and also another test fails when I don't clone.

Reply:
nah, you'll see a lot of "JSON.parse(JSON.stringify(" across our tests for the exact same reason. ;)

Reply:
FYI we do have `lodash.clonedeep` in our dependencies now which does this in a better way preserving `undefined`, `NaN`, classes, etc if you need it

```js
const cloneDeep = require('lodash.clonedeep');
console.log(cloneDeep({foo: undefined, bar: NaN, map: new Map([[1,2]])}))
// prints `{ foo: undefined, bar: NaN, map: Map { 1 => 2 } }`
```